ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -1270,9 +1270,6 @@ public XLHyperlink Hyperlink
         {
             get
             {
-                if (_hyperlink == null)
```

Comment:
We definitely want to keep this. Removing this will break unit tests and people's code.